index.html

```diff
@@ -371,26 +353,34 @@ <h2>EcdsaKoblitzSignature2016</h2>
       <tbody>
         <tr>
           <td>Identifiers</td>
-          <td>EcdsaKoblitzSignature2016</td>
+          <td>EcdsaSecp256k1VerificationKey2019</td>
         </tr>
         <tr>
           <td>Status</td>
           <td>PROVISIONAL</td>
         </tr>
         <tr>
           <td>Authors</td>
-          <td>Harlan Wood, Manu Sporny</td>
+          <td>Orie Steele</td>
         </tr>
         <tr>
           <td>Specification</td>
-          <td><a href="https://w3c-dvcg.github.io/lds-koblitz2016/">Koblitz Signature Suite 2016</a></td>
+          <td><a href="https://w3c-dvcg.github.io/lds-ecdsa-secp256k1-2019/">ECDSA Secp256k1 Signature Suite 2019</a></td>
         </tr>
       </tbody>
     </table>
+
+    <pre class="example nohighlight" title="Example of EcdsaSecp256k1VerificationKey2019">{
+  "id": "did:example:123456789abcdefghi#keys-1",
+  "type": "EcdsaSecp256k1VerificationKey2019",
+  "controller": "did:example:123456789abcdefghi",
+  "expires": "2017-02-08T16:02:20Z",
+  "publicKeyHex": "02b97c30de767f084...263d29f1450936b71"
+}</pre>
   </section>
 
   <section>
-    <h2>EcdsaSecp256k1Signature2019</h2>
+    <h2>EcdsaKoblitzSignature2016</h2>
```

Comment:
I'm not entirely sure what to do with the `EcdsaKoblitzSignature2016` suite - it doesn't have a corresponding verification key example. Is it superceded be the `EcdsaSecp256k1VerificationKey2019` spec? Should it go into its own 'Deprecated Suites' section?

Reply:
Yes, superceded by EcdsaSecp256k1VerificationKey2019 ... we could deprecate it, or just drop it entirely... don't think anyone ever deployed it in anything more than a PoC...

Reply:
Yes, deprecated, could delete it.... probably just deprecate it.